File: java/yapay-gateway-client/src/main/java/br/com/yapay/gateway/model/AirlineData.java
package br.com.yapay.gateway.model;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class AirlineData {

	@SerializedName("dadosViagem")
	private AirlineTripData tripData;

	@SerializedName("dadosPassageiros")
	private List<PassengerData> passengerData;

	@SerializedName("dadosEscalas")
	private List<FlightRangeData> flightRange;

	public AirlineTripData getTripData() {
		return tripData;
	}

	public void setTripData(AirlineTripData tripData) {
		this.tripData = tripData;
	}

	public List<PassengerData> getPassengerData() {
		return passengerData;
	}

	public void setPassengerData(List<PassengerData> passengerData) {
		this.passengerData = passengerData;
	}

}

File: net/YapayGatewayLib/model/AirlineData.cs
using System;
using System.Collections.Generic;

namespace YapayGatewayLib.Model
{
    public class AirlineData
    {
        AirlineTripData tripData { get; set; }
        List<PassengerData> passengerData { get; set; }
        List<FlightRangeData> flightRange { get; set; }
    }
}
